pom.xml

```diff
@@ -93,31 +93,32 @@
 
         <!-- Third-party dependencies (sorted alphabetically) -->
         <!--<affinity.version>3.2.3</affinity.version>-->
+        <antlr4.version>4.9.3</antlr4.version>
         <avro.version>1.11.0</avro.version>
-        <aws.sdk.version>1.12.120</aws.sdk.version>
-        <classgraph.version>4.8.66</classgraph.version>
-        <grpc.version>1.34.0</grpc.version>
+        <aws.sdk.version>1.12.183</aws.sdk.version>
+        <classgraph.version>4.8.139</classgraph.version>
+        <grpc.version>1.43.0</grpc.version>
         <guava.version>30.1.1-jre</guava.version>
-        <hadoop.version>3.3.0</hadoop.version>
-        <jackson.version>2.12.1</jackson.version>
-        <jline.version>3.19.0</jline.version>
+        <hadoop.version>3.3.1</hadoop.version>
```

Comment:
Consider fully backporting https://github.com/hazelcast/hazelcast/pull/19631 - it reduces size of the distribution by ~200 MB.

Reply:
Good point. I did a cherry-pick. Take one more look, please.